Common/Data/Format/PNGLoad.h

```diff
@@ -9,4 +12,20 @@ int pngLoad(const char *file, int *pwidth,
 int pngLoadPtr(const unsigned  char *input_ptr, size_t input_len, int *pwidth,
             int *pheight, unsigned char **image_data_ptr);
 
-#endif  // _PNG_LOAD_H
+// PNG peeker - just read the start of a PNG straight into this struct, in order to
+// look at basic parameters like width and height. Note that while PNG is a chunk-based
+// format, the IHDR chunk is REQUIRED to be the first one, so this will work.
```

Comment:
Well, as long as it's not [CgBI](http://iphonedevwiki.net/index.php/CgBI_file_format), which it probably won't be.

-[Unknown]

Reply:
Yeah, I can't be bothered to deal with that, heh.